content/intro-to-storybook/angular/en/get-started.md

```diff
@@ -14,44 +14,262 @@ Storybook runs alongside your app in development mode. It helps you build UI com
 We’ll need to follow a few steps to get the build process set up in your environment. To start with, we want to use the [@angular/cli](https://cli.angular.io/) to setup our build system, and enable [Storybook](https://storybook.js.org/) and [Jest](https://facebook.github.io/jest/) testing in our created app. Let’s run the following commands:
 
 ```bash
-# Create our application (using less as the style pre processor):
-npx -p @angular/cli ng new taskbox --style less
+# Create our application:
+npx -p @angular/cli ng new taskbox --style css
+
 cd taskbox
 
 # Add Storybook:
 npx -p @storybook/cli sb init
 ```
 
-We can quickly check that the various environments of our application are working properly:
+### Setup Jest with Angular
+
+We have two out of three modalities configured in our app, but we still need one, we need to setup [Jest](https://facebook.github.io/jest/) to enable testing.
+
+Run the following command:
 
 ```bash
-# Run the test runner (Karma) in a terminal (we will add Jest along the way):
-yarn test
+npm install -D jest @types/jest jest-preset-angular@7.1.1 @testing-library/angular @testing-library/jest-dom @babel/preset-env @babel/preset-typescript
+```
 
-# Start the component explorer on port 6006:
-yarn run storybook
+Create `src/jest-config` folder and inside some files will be added.
 
-# Run the frontend app proper on port 4200:
-yarn start
+Start by creating a file called `globalMocks.ts` with the following:
+
+```typescript
+const mock = () => {
+  let storage = {};
+  return {
+    getItem: key => (key in storage ? storage[key] : null),
+    setItem: (key, value) => (storage[key] = value || ''),
+    removeItem: key => delete storage[key],
+    clear: () => (storage = {}),
+  };
+};
+
+Object.defineProperty(window, 'localStorage', { value: mock() });
+Object.defineProperty(window, 'sessionStorage', { value: mock() });
+Object.defineProperty(window, 'getComputedStyle', {
+  value: () => ['-webkit-appearance'],
+});
 ```
 
-<div class="aside">
-  NOTE: If <code>yarn test</code> throws an error, you may need to install <code>watchman</code> as advised in <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">this issue</a>.
-</div>
+Followed with a new file called `setup.ts` with the following:
+
+```typescript
+import 'jest-preset-angular';
+import './globalMocks';
+
+Object.defineProperty(global, 'Promise', { writable: false, value: global.Promise });
+```
+
+Then create a new file called `styleMock.js` with the following:
+
+```javascript
+module.exports = {};
+```
+
+Also `fileMock.js` with the following:
+
+```javascript
+module.exports = 'file-stub';
+```
+
+Now in the root of your project folder, create a file called `babel.config.js` with the following content:
+
+```javascript
+module.exports = function(api) {
+  const presets = [
```

Comment:
```suggestion
  api.cache(true);
  ....
```

Hi, I've been following the guide in your PR because I had a lot of problems getting the current version to work. So far so good with this new version, except I also ran into the problem with the Babel configuration. I noticed below you've already addressed a discussion about that, however the config presented here still doesn't include the 'api.cache(true);' which tripped me up when I came to run the application.

(P.S. In your resolution below you mentioned you'd updated it in the repo, but I'm not sure if that meant you updated it in this document or in a repo with the complete example app source code. I'm new to GitHub so I'm not sure if I'm somehow missing the update here?)

Reply:
@alexanderdodd thank you for catching that, i need to circle back to this and go over it again as with the release of 5.3 some significant changes were introduced and i need to check them and adjust the documentation accordingly. The repo i mentioned in the pull request description is the one used for the tutorial. It contains allmost all of the implementation. The only thing missing is the test section, the `storybook-chromatic package` is there but it was being run locally. I'm still tinkering with that section to be applied not only with this version of the tutorial but to the remainder frameworks

Reply:
@jonniebigodes Great! I'm not sure if I missed it, but do you link out to the repo with the implementation in it anyway in the guides? If not, that would be really handy (I was trying to find such a thing when struggling with the version currently up on the website because it would've helped to have double checked the code and dependencies there).

Reply:
Ohh,sorry,  now i understand what you meant. In the conclusion there is a link to the [Storybook code repo](https://github.com/chromaui/learnstorybook-code), and that repo should have some branches with the code base used. But to be rather honest i haven't checked in a while and don't know if its currently updated to match the new release. In my todo list i had a item to check that code repo and see if i can make a pull request to add/update the codebase but that was put on the "back burner" (pardon the bad pun) as i kept working on the documentation. I'll have to check that out when i'm done with the documentation

Reply:
We try to keep the code repos up to date. You'll find the different frameworks on different branches of the linked repos.

Reply:
@alexanderdodd i'm marking this as resolved, as with the latest commit i've updated that code snippet and with that prevent a error that will pop up when you run Storybook. As it seems that babel is becoming a bit more "agressive" (pardon the bad pun) with their caching mechanism and leading to the developers to be more considerate on what strategy they should use.